plugins/arElasticSearchPlugin/lib/model/arElasticSearchInformationObject.class.php

```diff
@@ -198,4 +206,9 @@ public static function setBoostValues(&$fields, $cultures)
 
     self::addBoostValuesToFields($fields, $i18nBoostFields, $nonI18nBoostFields);
   }
+
+  public function getErrors()
```

Comment:
This doesn't seem related to the $skip code.

Reply:
This just allows search errors to be fetched by an external script. I forgot that I didn't include a link to an example script in this PR. My bad.